packages/userprofile/android/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
The `.idea` folder is deleted on staging branch, can we merge the latest staging changes to your fork repo/branch?